lua/ulx/modules/sh/cfc_brazil.lua

```diff
@@ -0,0 +1,138 @@
+local CATEGORY_TYPE = "Teleport"
+local MIN_X, MIN_Y, MIN_Z = -16384, -16384, -16384
+local MAX_X, MAX_Y, MAX_Z = 16384, 16384, 16384
+local MAX_TRIES = 20
+local MAP_POSITION_DATA = {
+    ["gm_bigcity"] = {
+        centerZ = 14500 / 2 - 11400,
+        playableHeight = 14500 / 2,
+        fallback = Vector( 9192, 10270, -10897 ),
+        offsetX = 900,
+        offsetY = 1900
+    },
+    ["gm_bigcity_improved"] = {
+        centerZ = 14500 / 2 - 11400,
+        playableHeight = 14500 / 2,
+        fallback = Vector( 9192, 10270, -10897 ),
+        offsetX = 900,
+        offsetY = 1900
+    },
+    ["gm_bigcity_improved_lite"] = {
+        centerZ = 14500 / 2 - 11400,
+        playableHeight = 14500 / 2,
+        fallback = Vector( 9192, 10270, -10897 ),
+        offsetX = 900,
+        offsetY = 1900
+    },
+    ["gm_construct"] = {
+        centerZ = 10500 / 2 - 500,
+        playableHeight = 10500 / 2,
+        fallback = Vector( -2069, -3316, -252 ),
+        offsetX = 1000,
+        offsetY = 500
+    },
+    ["gm_novenka"] = {
+        centerZ = 12800 / 2 - 2050,
+        playableHeight = 12800 / 2,
+        fallback = Vector( 9760, -13800, -700 ),
+        offsetX = 800,
+        offsetY = 500
+    },
+    ["gm_bluehills_test3"] = {
+        centerZ = 14400 / 2 - 1100,
+        playableHeight = 14400 / 2,
+        fallback = Vector( -8750, -8500, -1084 ),
+        offsetX = 500,
+        offsetY = 500
+    }, 
+    ["gm_flatgrass"] = {
+        centerZ = 28200 / 2 - 12800,
+        playableHeight = 28200 / 2,
+        fallback = Vector( -700, 100, -12764 ),
+        offsetX = 250,
+        offsetY = 500
+    }, 
+    ["gm_functional_flatgrass"] = {
+        centerZ = 28600 / 2 - 16300,
+        playableHeight = 28600 / 2,
+        fallback = Vector( 10500, -9400, -16252 ),
+        offsetX = 3500,
+        offsetY = 3500
+    }, 
+    ["gm_genesis"] = {
+        centerZ = 27600 / 2 - 15300,
+        playableHeight = 27600 / 2,
+        fallback = Vector( 6800, 6500, -14974 ),
+        offsetX = 600,
+        offsetY = 600
+    }
+}
+
+local function planTrip( ply )
```

Comment:
Not overly fond of this function name, it implies it returns some kind of "route" or "path" (trip), rather than just a position.
Perhaps something like `getRandomMapPos` or `generateMapPos` or whatever you might think is more fitting. Not hard fixed on anything.